Formula/g/gtk-vnc.rb

```diff
@@ -45,7 +48,7 @@ class GtkVnc < Formula
   end
 
   on_linux do
-    depends_on "libx11"
+    depends_on "gettext" => :build
```

Comment:
This is also a macOS dependency (see `on_macos` block above), so we can make the `:build` dependency unconditional